tests/system/test_query.py

```diff
@@ -422,6 +423,38 @@ def test_query_statistics(bigquery_client, query_api_method):
                 )
             ],
         ),
+        (
+            "SELECT @range_date",
+            "[2016-12-05, UNBOUNDED)",
```

Comment:
Is 'UNBOUNDED' some kind of type alias, or does the service return that as the literal string?  If so, should we convert UNBOUNDED -> None when build query parameter values?

Reply:
The service returns 'UNBOUNDED' as part of the literal string. 'UNBOUNDED' isn't an alias of any type, so it's safe to convert it into `None`.

Reply:
Thanks!  Its fine to punt on this aspect, I just wanted to clarify.